index.js

```diff
@@ -16,7 +15,8 @@ module.exports = options => {
 
   const load = (config) => {
     config.routes.forEach((route) => {
-      app.use(router(route, config));
+      config.route = route;
```

Comment:
I don't follow. this code is going to overwrite `config.route` for each route. what is it used for?